src/coreComponents/fieldSpecification/FieldSpecificationManager.cpp

```diff
@@ -246,28 +236,50 @@ void FieldSpecificationManager::validateBoundaryConditions( MeshLevel & mesh ) c
                                     "\nIf the simulation does not involve the SurfaceGenerator, check the content of the set `{}` in `{}`. \n",
                                     fs.getDataContext(), mapEntry.first, fs.getObjectPath() ) );
     }
-
     if( isFieldNameFound == 0 )
     {
-      std::ostringstream fieldNameNotFoundMessage;
-      std::string fieldNamePath =
-        GEOS_FMT( "\n{}: there is no {} named `{}` under the region `{}`.\n",
-                  fs.getWrapperDataContext( FieldSpecificationBase::viewKeyStruct::fieldNameString() ),
-                  FieldSpecificationBase::viewKeyStruct::fieldNameString(),
-                  fs.getFieldName(), fs.getObjectPath() );
-
-      fieldNameNotFoundMessage << fieldNamePath;
-      if( areAllSetsEmpty )
+      std::ostringstream errorMessageBuilder;
+      errorMessageBuilder << GEOS_FMT( "\n{}: there are no {} named `{}` under the region `{}`.\n",
```

Comment:
```suggestion
      errorMessageBuilder << GEOS_FMT( "\n{}: there are no field named `{}` under the region `{}`.\n",
```
rather than `[...] there are no fieldName named [...]`?